dagger/main.go

```diff
@@ -37,6 +36,7 @@ func (m *Dagger) TestChain(ctx context.Context, chain string, image string, sour
 		WithDirectory("/crosschain", source).
 		WithServiceBinding("node-service", nodeService).
 		WithEnvVariable("PRIVATE_KEY", privateKey).
-		WithExec([]string{fmt.Sprintf("go test crosschain/dagger/dagger_test.go -chain=%s", chain)}).
+		WithExec([]string{"sh", "-c", fmt.Sprintf("cd crosschain/ci/ && go test -run TestBalance --chain %s --foundAmount %s --rpc http://node-service:10000", chain, foundAmount)}).
+		WithExec([]string{"sh", "-c", fmt.Sprintf("cd crosschain/ci/ && go test -run TestTransfer --toAddress %s --transferedAmount %s --chain %s --foundAmount %s --rpc http://node-service:10000", toAddress, transferedAmount, chain, foundAmount)}).
```

Comment:
helpful to change these so `sh` isn't used, will make the dagger trace more clear.